base/applications/cmdutils/whoami/lang/it-IT.rc

```diff
@@ -36,22 +36,21 @@ BEGIN
     IDS_UNKNOWN_DESCRIPTION "???"
     IDS_STATE_ENABLED "Abilitato"
     IDS_STATE_DISABLED "Disabilitato"
-    IDS_ERROR_UPN "ERRORE: Impossibile trovare il nome dell'utente principale (UPN) perchè l'utente attualmente registrato\nnon è un dominio utente.\n"
-    IDS_ERROR_FQDN "ERRORE: Impossibile trovare il nome distinto pienamente qualificato (FQDN) perchè l'utente\nattualmente registrato non è un dominio utente.\n"
-    IDS_ERROR_VALUEXPECTED "ERRORE: Sintassi non valida. È previsto un valore per '/fo'.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_VALUENOTALLOWED "ERRORE: Sintassi non valida. '%s' valore non autorizzato per l'opzione '/fo'.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_1TIMES "ERRORE: Sintassi non valida. L'uso dell'opzione '%s' non è autorizzato più\n di una volta.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_INVALIDSYNTAX "ERRORE: Sintassi non valida.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_INVALIDARG "ERRORE: Argomento o opzione non valida - '%s'.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
-    IDS_ERROR_NH_LIST "ERRORE: L'opzione /NH non può essere usata in formato LIST.\nInserisci ""WHOAMI /?"" per vedere le informazioni dell'uso.\n"
+    IDS_ERROR_UPN "ERRORE: Impossibile trovare l'User Principal Name (UPN) perchè l'utente attualmente registrato\nnon è un dominio utente.\n"
```

Comment:
In my opinion older translation of this was better. UPN could be translated fine. But I suppose that also UPN need to be converted with italian world.

Reply:
I also thought of this but later on I doubt if these translated words did make any sense, technically speaking (the patch is actually provided by @SebastianoPistore). 

Undoubtedly, most (if not all) of English acronyms can be translated as much as we want but the problem lies down on localisation. Most of English IT words and labels are left un-translated for the sake of originality so I think it is better to keep it _as-is_.

Reply:
Yes. You're right. And this make sense becusa is a "pro" utility. I think that transaltion is needed more when we are providing an application for an unexperienced user.